.github/workflows/validate.yml

```diff
@@ -28,9 +28,11 @@ jobs:
         with:
           path: node_modules
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-modules-
 
       - name: Install packages
-        run: yarn install
+        run: yarn install --frozen-lockfile
```

Comment:
why is this being added?

Reply:
this is to make it reproducible for Cypress/action

Reply:
This is fair - frozen-lockfile means CI should be using the exact same dep versions as what we are using to develop.